salt/scripts.py

```diff
@@ -46,9 +46,24 @@ def _handle_signals(client, signum, sigframe):
         hardcrash = client.options.hard_crash
     except (AttributeError, KeyError):
         hardcrash = False
+
+    exit_msg = '\nExiting gracefully on Ctrl-c'
```

Comment:
This is not strictly true. This handler is registered to sigint and sigterm so anything that generates one of those will hit this code, including a user running a `kill` command.